context.go

```diff
@@ -499,6 +499,13 @@ func (ctx *Context) ServeFile(file string, names ...string) {
 	ctx.Resp.Header().Set("Expires", "0")
 	ctx.Resp.Header().Set("Cache-Control", "must-revalidate")
 	ctx.Resp.Header().Set("Pragma", "public")
+
+	if ctx.Resp.Header().Get("Content-Encoding") != "" {
+		f, _ := os.Open(name)
```

Comment:
Actually, just found out we can directly use `https://gowalker.org/os#Stat`, and set the header only if `err == nil`.